packages/react-core/src/demos/examples/Page/PageStickySectionBreadcrumb.tsx

```diff
@@ -0,0 +1,310 @@
+            <ToolbarItem visibility={{ default: 'hidden', md: 'hidden', lg: 'visible' }}>
+              <ApplicationLauncher
+                onSelect={onAppLauncherSelect}
+                onToggle={onAppLauncherToggle}
+                isOpen={isAppLauncherOpen}
+                items={appLauncherItems}
+              />
+            </ToolbarItem>
+            <ToolbarItem>
+              <Button aria-label="Settings" variant={ButtonVariant.plain} icon={<CogIcon />} />
+            </ToolbarItem>
+            <ToolbarItem>
+              <Button aria-label="Help" variant={ButtonVariant.plain} icon={<QuestionCircleIcon />} />
+            </ToolbarItem>
+          </ToolbarGroup>
+          <ToolbarItem visibility={{ default: 'hidden', md: 'visible', lg: 'hidden' }}>
+            <Dropdown
+              isPlain
+              position="right"
+              onSelect={onKebabDropdownSelect}
+              toggle={<KebabToggle onToggle={onKebabDropdownToggle} />}
+              isOpen={isKebabDropdownOpen}
+              dropdownItems={kebabDropdownItems}
+            />
+          </ToolbarItem>
+          <ToolbarItem visibility={{ md: 'hidden' }}>
+            <Dropdown
+              isPlain
+              position="right"
+              onSelect={onFullKebabDropdownSelect}
+              toggle={<KebabToggle onToggle={onFullKebabDropdownToggle} />}
+              isOpen={isFullKebabDropdownOpen}
+              dropdownItems={fullKebabDropdownItems}
+            />
+          </ToolbarItem>
+        </ToolbarGroup>
+        <ToolbarItem visibility={{ default: 'hidden', md: 'visible' }}>
+          <Dropdown
+            isFullHeight
+            onSelect={onDropdownSelect}
+            isOpen={isDropdownOpen}
+            toggle={
+              <DropdownToggle icon={<Avatar src={imgAvatar} alt="Avatar" />} onToggle={onDropdownToggle}>
+                Ned Username
+              </DropdownToggle>
+            }
+            dropdownItems={userDropdownItems}
+          />
+        </ToolbarItem>
+      </ToolbarContent>
+    </Toolbar>
+  );
+
+  const masthead = (
+    <Masthead>
+      <MastheadToggle>
+        <PageToggleButton variant="plain" aria-label="Global navigation">
+          <BarsIcon />
+        </PageToggleButton>
+      </MastheadToggle>
+      <MastheadMain>
+        <MastheadBrand>
+          <Brand
+            widths={{ default: '180px', md: '180px', '2xl': '220px' }}
+            src="/assets/images/logo__pf--reverse--base.png"
+            alt="Fallback patternFly default logo"
+          >
+            <source media="(min-width: 768px)" srcSet="/assets/images/logo__pf--reverse-on-md.svg" />
+            <source srcSet="/assets/images/logo__pf--reverse--base.svg" />
+          </Brand>
+        </MastheadBrand>
+      </MastheadMain>
+      <MastheadContent>{headerToolbar}</MastheadContent>
+    </Masthead>
+  );
+
+  const pageNav = (
+    <Nav onSelect={onNavSelect} aria-label="Nav">
```

Comment:
```suggestion
    <Nav onSelect={onNavSelect}>
```

I _think_ we want this `aria-label` to be "Global", otherwise the rotor reads this as "Nav navigation". "Global" is the, so if we want to update, you could remove this entirely. https://github.com/patternfly/patternfly-react/blob/38e7a38cfcbb11b376302e90c1ac9df3a3f00711/packages/react-core/src/components/Nav/Nav.tsx#L162

If so, we should update the main dashboard nav label, too - https://github.com/patternfly/patternfly-react/blob/7f16b58b27382a7007bd99acfdb3b3fe4f6009b7/packages/react-core/src/demos/examples/DashboardWrapper.js#L73

This also matches the `<PageToggleButton>`'s `aria-label`. Those labels should probably match. https://github.com/patternfly/patternfly-react/blob/7f16b58b27382a7007bd99acfdb3b3fe4f6009b7/packages/react-core/src/demos/examples/DashboardHeader.js#L191